bin/resty

```diff
@@ -130,7 +130,15 @@ my $conns = $conns_num || 64;
 my @nameservers;
 
 if (!$resolve_ipv6) {
-    unshift @nameservers, "ipv6=off";
+    # no error check to let the later exec() fail.
+    # worst case: ipv6=off is not added to the nginx
+    # conf, but the worker won't spawn anyways...
+    my $stderr = `$nginx_path -V 2>&1 1>/dev/null`;
+    #warn $stderr;
+
+    if (index($stderr, "--with-ipv6") != -1) {
```

Comment:
Maybe we could test this from the Perl script then? Opening a connection over IPv6 and see if we succeed?

Ideally, wouldn't it make sense for Nginx 1.11.5+ to ignore the `ipv6=off` flag from the `resolver` directive when support for IPv6 is disabled?